egs/must_c/mt1/conf/tuning/train_pytorch_transformer.yaml

```diff
@@ -0,0 +1,42 @@
+# network architecture
+# encoder related
+elayers: 6
+eunits: 2048
+# decoder related
+dlayers: 6
+dunits: 2048
+# attention related
+adim: 256
+aheads: 4
+tie-src-tgt-embedding: false
+tie-classifier: false
+
+# label smoothing
+lsm-weight: 0.1
+
+# minibatch related
+batch-size: 64  # for BPE
+maxlen-in: 100  # if input length  > maxlen-in, batchsize is automatically reduced
+maxlen-out: 100 # if output length > maxlen-out, batchsize is automatically reduced
+
+# optimization related
+sortagrad: 0 # Feed samples from shortest to longest ; -1: enabled for all epochs, 0: disabled, other: enabled for 'other' epochs
+opt: noam
```

Comment:
Question: what are noam and transformer-lr for? Is vanilla Adam not good?
In Transformer-NMT, I think that we usually use Adam with warmup https://github.com/pytorch/fairseq/tree/master/examples/scaling_nmt

Reply:
Noam in ESPnet means Adam with lr scheduling based on the original Transformer paper.

Reply:
And also the hyperparameter such as beta is modified.

Reply:
I see. Do you think adding a description for that is a good idea?

Reply:
good idea! maybe, we should add this in the main readme. I'll do it with another PR.